nilearn/plotting/img_plotting.py

```diff
@@ -595,6 +600,168 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     return display
 
 
+def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
+                    threshold=None, linewidths=2.5, cut_coords=None,
+                    output_file=None, display_mode='ortho',
+                    figure=None, axes=None, title=None, annotate=True,
+                    draw_cross=True, black_bg='auto', dim=False,
+                    cmap=plt.cm.gist_rainbow, vmin=None, vmax=None,
+                    alpha=0.5, **kwargs):
+
+    """ Plot the probabilistic atlases onto the anatomical image
+        by default MNI template
+
+        Parameters
+        ----------
+        maps_img: Niimg-like object or the filename
+            4D image of the probabilistic atlas maps
+        anat_img : Niimg-like object
+            See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
+            The anatomical image to be used as a background. If None is
+            given, nilearn tries to find a T1 template.
+        view_type: {'auto', 'contours', 'filled_contours', 'continuous'}, optional
+            By default view_type == 'auto', which means maps are overlayed as
+            contours if number of maps to display are more or
+            overlayed as continuous colors if number of maps are less.
+            If view_type == 'contours', maps are overlayed as contours
+            If view_type == 'filled_contours', maps are overlayed as contours
+            along with color fillings inside the contours.
+            If view_type == 'continuous', maps are overlayed as continous
+            colors irrespective of the number maps.
+        threshold: None, a str or a number, list of either str or number, optional
+            If threshold is a string it must finish with a percent sign,
+            e.g. "25.3%", or if it is a number it should be real numbers.
```

Comment:
To be more precise, I would write.

e.g. "25.3%", and it is a percentile. Or if it is a number, if should be a real number, in which case it is the value to threshold at.